docs/resources/cbh_instance.md

```diff
@@ -69,203 +68,55 @@ The following arguments are supported:
-
-* `flavor_id` - (Optional, String) Specifies the product ID of the bandwidth associated with.
-
-<a name="CBHInstance_RootVolume"></a>
-The `RootVolume` block supports:
-
-* `type` - (Optional, String) Specifies the type of volume.
-
-* `size` - (Optional, String) Specifies the size of the root volume, unit is GB.
-
-* `extend_param` - (Optional, String) Specifies the info of the volume.
-
-<a name="CBHInstance_DataVolume"></a>
-The `DataVolume` block supports:
-
-* `type` - (Optional, String) Specifies the type of volume.
-
-* `size` - (Optional, String) Specifies the size of the data volume, unit is GB.
-
-* `extend_param` - (Optional, String) Specifies the info of the volume.
-
-<a name="CBHInstance_ProductInfo"></a>
-The `ProductInfo` block supports:
-
-* `product_id` - (Optional, String) Specifies the ID of the product.
-
-* `cloud_service_type` - (Optional, String) Specifies the type of the CBH service.
-
-* `resource_type` - (Optional, String) Specifies the type of the CBH resource.
-
-* `resource_spec_code` - (Optional, String) Specifies the flavor of the CBH service.
-
-* `availability_zone` - (Optional, String) Specifies the availability zone name.
-
-* `resource_size_measure_id` - (Optional, String) Specifies the resource capacity measurement ID.
-
-* `resource_size` - (Optional, String) Specifies the size of the resource capacity.
-
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The resource ID.
 
-* `publicip_id` - Indicates the ID of the elastic IP.
-
-* `exp_time` - Indicates the expire time of the instance.
-
-* `start_time` - Indicates the start time of the instance.
-
-* `release_time` - Indicates the release time of the instance.
-
-* `instance_id` - Indicates the server id of the instance.
-
 * `private_ip` - Indicates the private ip of the instance.
 
-* `task_status` - Indicates the task status of the instance.
-
 * `status` - Indicates the status of the instance.
 
-* `update` - Indicates whether the instance image can be upgraded.
-
-* `instance_key` - Indicates the ID of the instance.
-
-* `resource_id` - Indicates the ID of the resource.
-
-* `alter_permit` - Indicates whether the front-end displays the capacity expansion button.
-
-* `bastion_version` - Indicates the current version of the instance image.
-
-* `new_bastion_version` - Indicates the latest version of the instance image.
-
-* `instance_status` - Indicates the status of the instance.
-
-* `description` - Indicates the type of the bastion.
+* `version` - Indicates the current version of the instance image.
 
 ## Timeouts
 
 This resource provides the following timeouts configuration options:
 
-* `create` - Default is 30 minute.
-* `delete` - Default is 10 minute.
+* `create` - Default is 60 minutes.
+* `delete` - Default is 30 minutes.
 
 ## Import
 
-The cbh instance can be imported using the `id`, e.g.
+The CBH instance can be imported using the `id`, e.g.
```

Comment:
some parameters can not be imported, please add a note here.

Reply:
done